README.md

```diff
@@ -509,9 +509,39 @@
     [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).
 
 * **What is the difference between a constructor and a method?** 
-    - The name of the constructor is same as that of the class name, whereas the name of the method can be anything.
-    - There is no return type of a constructor.
-    - When you make an object of a class, then the constructor of that class will be called automatically. But for methods, we need to call it explicitely.
+    - Constructor and a method they both run a block of code but the difference is in calling them.
+    - 1) Constructor gets called automatically when we create an object of that class, we cannot call it explicitly
+         though we can call parent class constructor using 'super' keyword.
+    - 1) We call method directly using their name.
+    - 2) Constructor Syntax - 
+              
+              public class SomeClass{
+              
+              SomeClass(parameter_list){ 
+              ...
+              } 
+              
+              ...
+              
+              }
+
+Note: 
+Constructor name is the same as class name
+Constructor does not have return type
+
+    -2) Method Syntax - 
+    
+              public class SomeClass{
+              
+                public void someMethod(){
+                  ...
+                }
+                // call method
+                someMethod()
+              }
+              
+    - Name of the method can be anything.
+    - When you make an object of a class, then the constructor of that class will be called automatically. But for methods, we need to         call it explicitely.
```

Comment:
Do a little bit of formatting:
1. please formate the changes such as use bullet points and not 1, 2, 3....
2. remove extra space from the constructor and function snippet

Please preview the file after changing the text..

Reply:
Okay formatting it